### PR TITLE
Sprint wk42

### PR DESCRIPTION
**Pull request by penninghlhd**

### CODE REVIEW

File: library/src/main/cpp/libenc/libenc.cc [hunk 1]
```diff
@@ -225,15 +226,16 @@ YUV420_888toI420(uint8_t *src_y, jint y_stride,
             i++;
         }
     }
+
 //    ret = Android420ToI420(src_y, y_stride,
```
Comment (daanishr) [inline library/src/main/cpp/libenc/libenc.cc:230]: Unused code, comment not used or in progress?

Comment (daanishr) [inline library/src/main/cpp/libenc/libenc.cc]: Unused code, comment not used or in progress?

**Review by daanishr — APPROVED** [review]
Looks good

**Review by daanishr — APPROVED** [review]

File: library/src/main/cpp/libenc/libenc.cc (hunk 1 above)
Comment (penninghlhd) [reply library/src/main/cpp/libenc/libenc.cc:230]: Will remove it

**Pull request merged**